controls/V-219167.rb

```diff
@@ -118,5 +118,20 @@
   tag fix_id: 'F-20891r304830_fix'
   tag cci: ['V-100561', 'SV-109665', 'CCI-000050']
   tag nist: ['AC-8 b']
-end
 
+  if !package('gdm3').installed?
```

Comment:
please use `unless` rather than `if !`

Reply:
Also, if you enable 'maintainers edit' for PRs I could help a bit as well

Reply:
`unless package('gdm3').installed?`
    foo
  else
    bar
  end

Reply:
Updated unless, brought in the gsub whitespace stripping from RHEL/CIS, and bumped threshold in [7089ab6](https://github.com/mitre/canonical-ubuntu-18.04-lts-stig-baseline/pull/24/commits/7089ab60ee17cd5fdb004327c387f09a6db2a745)